v2/router.go

```diff
@@ -29,15 +29,15 @@ import (
 // TODO: Until golang/mock#4 is fixed, imports in the generated code have to
 // be fixed by hand. They use vendor/* import paths rather than direct.
 
-// Procedure specifies a single handler registered in the RouteTable.
-type Procedure struct {
+// TransportProcedure specifies a single transport-level handler registered in the RouteTable.
+type TransportProcedure struct {
 	// Name of the procedure.
 	Name string
 
 	// Service or empty to use the default service name.
 	Service string
 
-	// TransportHandlerSpec specifying which handler and rpc type.
+	// HandlerSpec specifying which handler and rpc type.
 	HandlerSpec TransportHandlerSpec
 
 	// Encoding of the handler.
```

Comment:
With this change, does this serve any purpose?

Reply:
I'm fixing a comment I accidentally changed in an older commit.

Reply:
Sorry, I should have clarified - I was commenting on the `Encoding` field.